.github/workflows/process_every_payload.yaml

```diff
@@ -0,0 +1,14 @@
+name: Process BIP PR
```

Comment:
why need this?

Reply:
The naame or the action.

The action is needed to trigger the reports when a PR is opened (under the on conditions listed in this file).

The Name probs isn't needed, but it makes it easier to look at.  What this does is runs on PR and does bip processing.